captalize_decorate.rb

```diff
@@ -0,0 +1,7 @@
+require './decorate'
+
+class CapitalizeDecorator < Decorate
+  def correct_name
+    decorate.correct_name.captalize
```

Comment:
- [ ] There is no variable named `decorate`. What you should have is `@nameable` as inherited from the Decorator class. Kindly fix that.

- [ ] You also spelt "capitalize" wrong.

  ```
  @nameable.correct_name.capitalize
  ```